views/database/show.erb

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+  <h2>Database: <%= @db.name %> -
+    <a href="/db/<%= @db.id %>/edit">edit</a>
+  </h2>
+  
+  <p>
+    Project Id: <%= @db.project_id %> <br>
+    Type: <%= @db.type %> <br>
+    Server: <%= @db.server %> <br>
+    Active: <%= @db.active %><br>
+  </p>
+  <table class="table">
+      <thead>
+        <tr>
+          <th>Project Id</th>
+          <th>Type</th>
+          <th>Server</th>
+          <th>Active</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% for @project in @projects %>
```

Comment:
I think there should only ever be one project for a database, since the model relation is one project to many dbs, so I am not sure what this whole table is doing here